tqp/newrelic.py

```diff
@@ -27,6 +27,9 @@ def _hook_send_newrelic_event(self, payload, success):
                 "topic": payload["topic"],
                 "queue_name": self.queue_name,
                 "success": str(success),
+                "attributes": payload["meta"].get("attributes", {})
+                if payload["meta"]
+                else {},
```

Comment:
```suggestion
                "attributes": (payload["meta"].get("attributes", {})
                if payload["meta"]
                else {}),
```

so black doesn't screw this up